compiler/src/dmd/backend/arm/cod3.d

```diff
@@ -1511,10 +1511,11 @@ printf("offset: %lld localsize: %lld REGSIZE*2: %d\n", offset, localsize, REGSIZ
             case FL.func:
             case FL.code:
             case FL.unde:
+            case FL.block:
                 break;
 
             default:
-                if (0) printf("FL: %s\n", fl_str(c.IFL1));
+                if (1) printf("FL: %s\n", fl_str(c.IFL1));
```

Comment:
are you intending to actually turn on the debug printf?

Reply:
My master branch has a mismatch with the repository which is why there are 2 commits here. My attempts to resolve it have failed. My plan is to get this merged, then do a hard reset on my master.